examples/c/ksyscall.bpf.c

```diff
@@ -0,0 +1,50 @@
+#include "vmlinux.h"
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_tracing.h>
+#include <bpf/bpf_core_read.h>
+
+#define TASK_COMM_LEN 16
+
+SEC("ksyscall/tgkill")
+int BPF_KSYSCALL(tgkill_entry, pid_t tgid, pid_t tid, int sig) 
+{
+
+    char comm[TASK_COMM_LEN];
+    __u32 caller_pid = bpf_get_current_pid_tgid() >> 32;
```

Comment:
formatting seems off, please double check that you are not mixing spaces and tabs, use tabs consistently

also, please keep empty line between variable declarations and the rest of the code